### PR TITLE
[PLATFORM-446] fix(add): send credentials when adding resources

### PR DESCRIPTION
**Pull request by raulb**
# Description of change

Fixes https://meroxa.atlassian.net/browse/PLATFORM-446

# Type of change

- [ ]  New feature
- [x]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

# How was this tested?

- [x]  Unit Tests
- [ ]  Tested in staging

# Demo

**Before this pull-request**

```
❯ .m add resource webhook --type url -u https://webhook.site/77d8104b-2dca-40d4-a8c6-cf349f2f7363
Adding url resource...
Error: resource improperly configured
details:
	"credentials": ["must be provided"]
```

**After this pull-request**

```
❯ .m add resource webhook --type url -u https://webhook.site/77d8104b-2dca-40d4-a8c6-cf349f2f7363
Adding url resource...
url resource with name webhook successfully added!
```

### CODE REVIEW

**Comment by ahmeroxa** [comment]
Acceptance tests are failing. Do we need to ship a `platform-api` fix first?

File: cmd/add_resource_test.go [hunk 1]
```diff
@@ -75,12 +75,13 @@ func TestAddResourceExecution(t *testing.T) {
...
 
+	var emptyCreds meroxa.Credentials
+
 	r := meroxa.CreateResourceInput{
 		Type:        "postgres",
 		Name:        "",
 		URL:         "https://foo.url",
-		Credentials: nil,
-		Metadata:    nil,
+		Credentials: &emptyCreds,
```
Comment (mwoods79) [inline cmd/add_resource_test.go:84]: This will be a pointer to nil, is this what you want?

Comment (raulb) [reply cmd/add_resource_test.go:84]: @mwoods79 yeah... https://github.com/meroxa/meroxa-go/blob/master/resource.go#L34

**Comment by raulb** [comment]
@ahmeroxa looks like something we need to fix in https://github.com/meroxa/acceptance. Will look into it.

**Comment by raulb** [comment]
We ended up not needing this in the CLI. For context, API already does a check on the existence of credentials in both URL and creds being sent out from the CLI ,but we needed to ad an exception to the HTTP type which is what we did in https://github.com/meroxa/platform-api/pull/321.

**Pull request closed without merging**